### ISSUE TITLE
UI skin preparation

### ISSUE THREAD

**Issue by follower46** [clockworkpi/launcher#64]
It would be nice if all of the UI pygame.Color parameters came from the `skin_manager`
I'd like to work on switching it over.

This change should have zero change to the visuals of the system (I'm not going to add any skins)

This will change these types of calls:
`pygame.Color(R,G,B)`
`component.Init("title",font,(R,G,B))`
`pygame.draw.shape(self._CanvasHWND,(R,G,B),param,0)`
`component.fill( (R,G,B))`

This change won't handle fonts or any other skin elements.
This is just the first step on getting all the UI's unified color-wise

**Comment by follower46** [comment]
After re-reading through the skin manager code, this may not be needed. lol

**Comment by follower46** [comment]
woah, this is actually a lot of work, lol

@cuu what do you think about changing the SkinManager class to be a singleton?
That way we won't have to pass the reference to every page which needs to be themed and it would greatly speed up getting the whole UI themeable

**Comment by follower46** [comment]
I created https://github.com/clockworkpi/launcher/pull/86 with a sample of it being used on the storage page.

If you like this I'll start converting all the pages over, if not let me know what you think is better :+1:

**Comment by follower46** [comment]
Awesome, I should have the code updated tomorrow.
Thank you for looking! :smile:

**Comment by follower46** [comment]
Ok, I completed the remaining items.
There were a few I didn't convert either because I had no way to test them or they aren't able to be called.

Here's a quick clip of my tests (I changed the UI to a yellow theme to verify the unskinned elements were converted): https://www.youtube.com/watch?v=9eh7dVp76To

Each of these pages had elements which were static which can now be skinned (including the launching dialog).
